### ISSUE TITLE
Unloading of Grape and Olive Harvesters

### ISSUE THREAD

**Issue by Zoochase** [Courseplay/Courseplay_FS22#1873]
When the harvester is full, have it back out of the orchard and wait for a trailer to unload in.  As soon as it is empty, it resumes in the same row.
Thanks for the consideration.

**Comment by rohne83** [comment]
Never tried those fruits and think I'll never will. But could this work with combine self-unload activated? Or if not, you could try if it already works with an unload course for AutoDrive (so handle harvester like a loading wagon or stone picker).

**Comment by Tensuko** [comment]
Combine selfunload is the planed way, but ofc it needs extra handling to get out of the lane, so this is something for much later.

**Comment by Zoochase** [comment]
Wouldn't my suggestion be a simpler temporary solution?

**Comment by Tensuko** [comment]
No, becouse "wait for a trailer to unload in" is the first step of make an automatic unloader for it.
And as soon as we are out of the row, there should be not much left to self unload and back to target.

Both take work and thats it -> patient, low priority, sry.

**Comment by Zoochase** [comment]
No worries.  Thank you for the explanation.
Z

**Comment by lcc1211** [comment]
Is AutoDrive replacing some functions of CoursePlay?

**Comment by fudoido** [comment]
> Is AutoDrive replacing some functions of CoursePlay?

compared to what? fs19 courseplay? the yes, autodrive is taking over all driving modes, leaving courseplay for field operations only. They are however working together and one mod can trigger the other mod. 
As an example, bale works in a field. you set autodrive course to go from field to bale sale. with a bale autoload trailer equipped, you make the vehicle wait INSIDE the field and turn ON the courseplay interaction button. When vehicle reaches the field waypoint, autodrive will pass control to courseplay, which detects the bales and picks them up. when trailer is full of bales, courseplay will pass control to autodrive again, which will then drive to bale sale point, unload bales and return to field,  and again when reaching field, will pass control to courseplay and repeat the operation until no more bales found.

If you mean replacing current fs22 versions functions, i dont think it is, both mods have the same functions they had from fs22 release, courseplay isa dding more functions even, but not replacing as far i am aware.

**Comment by lcc1211** [comment]
> > Is AutoDrive replacing some functions of CoursePlay?
> 
> compared to what? fs19 courseplay? then yes, autodrive is taking over all driving modes, leaving courseplay for field operations only. They are however working together and one mod can trigger the other mod. As an example, bale works in a field. you set autodrive course to go from field to bale sale. with a bale autoload trailer equipped, you make the vehicle wait INSIDE the field and turn ON the courseplay interaction button. When vehicle reaches the field waypoint, autodrive will pass control to courseplay, which detects the bales and picks them up. when trailer is full of bales, courseplay will pass control to autodrive again, which will then drive to bale sale point, unload bales and return to field, and again when reaching field, will pass control to courseplay and repeat the operation until no more bales found.
> 
> If you mean replacing current fs22 versions functions, i dont think it is, both mods have the same functions they had from fs22 release, courseplay is adding more functions even, but not replacing as far i am aware.

Was this because offloading by CoursePlay no longer worked or was this a preference thing?

**Comment by Tensuko** [comment]
If you ask about 'some funtions' no one can answer you.
What exactly do you want to know.
Beside that this issue is the wrong place for general talk and we have the discussion section for that.....

**Comment by lcc1211** [comment]
> If you ask about 'some funtions' no one can answer you. What exactly do you want to know. Beside that this issue is the wrong place for general talk and we have the discussion section for that.....

Yet I got an answer and I followed up with a specific that I also asked you directly once before, but as I expected from you, I got a sarcastic non-answer about why offloading wasn't part of CP anymore, and why must I use a new mod to do what CP was already doing. I've read so many of your responses to other people that come across as petulant, rude and arrogant, but we have endured your replies because we either needed help or desired answers.

**Comment by Tensuko** [comment]
I just give short and clear answers. Thats all :)
And the answer back then was we started all over again with courseplay. No Idea why got to bring it up again. And again we have a discussion part for this.
